src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -99,4 +121,15 @@ public static function checkForIllegalCsvPropertyKeys($propertyKey)
             throw new BadPropertyKeyException($propertyKey);
         }
     }
+
+    /**
+     * @param string $attributeName Attribute name to output in exception.
+     * @param string $attributeValue Attribute value to check if it exceeds character limit.
+     */
+    public static function checkValueNotExceedingCharacterLimit($attributeName, $attributeValue)
+    {
+        if (strlen($attributeValue) > self::CHARACTER_LIMIT) {
```

Comment:
Could be more restrictive than necessary: `strlen` counts bytes, `mb_strlen` counts characters. Now here I don't know by heart if the limit we have for attribute values is a byte limit or a character limit. Please check this and adapt accordingly!